README.md

```diff
@@ -2,96 +2,33 @@
 
 # Polkadot Secure Validator Setup
 
-This repo describes a potential setup for a Polkadot validator that aims to prevent
-some types of potential attacks.
+This repo describes a potential setup for a Polkadot validator that aims to
```

Comment:
We need a link here to the setup described in the reference document that the tool implements https://hackmd.io/QSJlqjZpQBihEU_ojmtR8g